src/Server_TEST.cc

```diff
@@ -1088,6 +1088,41 @@ TEST_P(ServerFixture, AddResourcePaths)
   }
 }
 
+/////////////////////////////////////////////////
+TEST_P(ServerFixture, SdfWithoutWorld)
+{
+  // Initialize logging
+  std::string logBasePath = common::joinPaths(PROJECT_BINARY_PATH, "tmp");
+  common::setenv(IGN_HOMEDIR, logBasePath);
+  std::string path = common::uuid();
+  ignLogInit(path, "test.log");
+
+  // Start server with model SDF file
+  ServerConfig serverConfig;
+  serverConfig.SetSdfFile(std::string(PROJECT_SOURCE_PATH) +
+      "/test/worlds/models/sphere/model.sdf");
```

Comment:
use `common::joinPaths`

Reply:
Done in [9059a2b](https://github.com/gazebosim/gz-sim/pull/1998/commits/9059a2b3073f4de18920775245ed66dba432a62b).